### PR TITLE
[gh-pages] [ZEPPELIN-2597] Updating offcial main website

### PR DESCRIPTION
**Pull request by soralee**
## What is this PR for?
This PR is for Improving more comfortable guidance for point of user view in the main zeppelin homepage. 

To sum up, `Technologies` section is consist of  3 interpreters most people search or want (already talking about this on the mailing list.) and, `Deployments` section describes single or multiple user setting.

Please check **TODO** list and Screenshots for detail. Plus, note that this PR is related #2371 PR. 
And if anyone suggests idea or give me feedback, it would be appreciated!

## What type of PR is it?
[Improvement | Bug fix]

## Todos
~~* [x] - change navigation bar items.~~
       ~~- add `Quick Start`~~
       ~~- move `Community` and add `Github` in here~~
~~* [x] - remove 'Apache Aeppelin' title in the blue section and change button.~~
     ~~- `Get Started` and `Download`~~
~~* [x] - add form and mouse event css for quick interpreter links section (Technologies).~~
~~* [x] - add form and css for user setting option (Deployments).~~
~~* [ ] - connect to each quick links. (@1ambda, could you help about this, please?)~~
~~* [x] - adding zeppelin log image in the blue section.~~

1) in blue section
- [x] add zeppelin logo in the blue section
- [x] fix to bug about misalign apache zeppelin text and icon
- [x] change `Get started` button hover color
- [x] connecting a link to `Get started` button

2) regarding to improve navbar
 - [x] apply style
 - [x] change border color
 - [x] change background color and text style
 - [x] fix : not fully hovered color
 - [x] fix :  when resizing window, drop-menu toggle text is appeared.
 - [x] fix : after using navigation bar in small window, and then drop-toggle place is misaligned in big window.

3) regarding to improve layout
 - [x] when resizing window
 - [x] change multi-user icon
 - [x] improve what's new section

4) in technologies section
- [x] set more bigger text in technologies
- [x] apply sql font
- [x] connecting quick start links

5)  in deployments section
- [x] change title for user setting option (single user, multi user)
- [x] connecting quick start links

## What is the Jira issue?
* [ZEPPELIN-2597](https://issues.apache.org/jira/browse/ZEPPELIN-2597)

## How should this be tested?
1. run `bundle exec jekyll serve --watch` in this branch
2. open `localhost:4000`

### 1. overall shape
image

### 2. changed navigation
peek 2017-06-01 14-37

### 3. Technologies
technologies

### 4. Deployments
deployments

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? Yes, this is

### CODE REVIEW

**Comment by 1ambda** [comment]
@soralee Thanks for updating official page. Could you add `[gh-pages]` prefix to the PR title?

**Comment by 1ambda** [comment]
regarding navbar-collapse button, it would be nice to modify `navbar-inverse. .navbar-toggle` classes style in style.css:L151 to

image

image

**Comment by 1ambda** [comment]
Setting border color to white would be better. 
For code, you can refer https://github.com/apache/zeppelin/pull/2355

### Before

image
 
### After 

image

**Comment by 1ambda** [comment]
Background colors and text styles can be improved. 

### Current

image

### After

image

### Code

For title, let's use `text-shadow: 1px 2px #353131` for better readability.

**Comment by 1ambda** [comment]
Layout in small browser can be improved too.

### Current

image

image

File: advertise.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+          <label class="content-text">
+            Local Spark, 6 Built-in visualizations, Display system, Dynamic form, Multiple backends are supported.<br/>
+            <a href="" class="user-button">LEARN MORE</a>
+          </label>
+        </p>
+      </div>
+      <div class="border col-md-6 col-sm-6">
+        <p class="panel-content-user">
+          <label>
+            <span class="glyphicon glyphicon-cog" style="margin:0 0 0 155px;font-size:33px;position:relative;"></span>
```
Comment (1ambda) [inline advertise.md]: Can we use `fa-users` for the **Multi-User** section? It's more explicit icon.

**Comment by 1ambda** [comment]
Can we improve the layout this section too? Currently, it looks like

image

**Comment by 1ambda** [comment]
Descriptions in the **TECHNOLOGIES** section is too small. It's hard to read.

image

**Comment by 1ambda** [comment]
If it's impossible to specific logos (e.g mysql, postres) due to license in the **SQL** button, IMO we can apply this kind of design. (font is not fancy, but we can improve.)

image

**Comment by 1ambda** [comment]
Also, icons and texts are slightly moved in a medium size browser.

### NORMAL: Large Size Browser

image

### PROBLEM: Medium Size Browser

image

**Comment by 1ambda** [comment]
Can we use a different color for hovered button?

image

**Comment by 1ambda** [comment]
Buttons in navbar are not fully hovered. Height should be 60. 
You can refer https://github.com/apache/zeppelin/pull/2355

### Current

image

### After 

image

**Comment by soralee** [comment]
Wow, @1ambda, thanks a lots for review!
I'll improve as your comments!

**Comment by Leemoonsoo** [comment]
How about

"For single user" -> "Single user"
"For multi user" -> "Multi user"

?

**Comment by soralee** [comment]
I'm improving navigation bar but, I found two bug in navigation bar.
so we should resolve those issues together.
1. when resizing window, drop-toggle is appeared. 
bug1

2. after using navigation bar in small window, and then drop-toggle place is misaligned in big window.
bug2

**Comment by soralee** [comment]
one more bug problem, if I escape `download` menu, color still leaves active color.
hover bug

**Comment by soralee** [comment]
Hi, finally I finished the above To-do list.
@1ambda @Leemoonsoo Could you confirm this, again?

For now, the quick links is connected with 0.8.0-SNAPSHOT documentation. @Leemoonsoo could you advice this, please?

**Comment by AhyoungRyu** [comment]
Thank you for great work! Especially big +1 for adding this section

screen shot 2017-06-04 at 11 27 48 am

I've always thought we need this kind of guide in first page of website.

I quickly look through the change and here are some thought in my head. 

### 1. Navbar
 - before 
screen shot 2017-06-04 at 11 20 39 am

 - after 
screen shot 2017-06-04 at 11 20 22 am

Adding "Quick Start" menu and switching menu order are really good idea I think. But not sure about making the font size bigger and adding too much shadow. So in my opinion, just below work will be enough.

    - Adding "Quick Start" menu
    - Switching menu order 


### 2. Main
I also agree this part doesn't have really good looking.
screen shot 2017-06-04 at 11 37 38 am

But not sure the new change is definitely better than current one.
screen shot 2017-06-04 at 11 38 22 am



I think it'll be better to update other parts expect "Technologies" & "Deployments" that you newly added in another PR step by step. That would be easier to get review/ consensus from other ppl and also easier to make it merged. I mean we can separate the work like below. 

```
1. update navbar
2. update main section
3. adding "Technologies" & "Deployment" section
4. minor styling 
....
```
What do you think? 
Thanks again for staring this work.

File: assets/themes/zeppelin/css/style.css [hunk 2]
```diff
@@ -88,32 +99,41 @@ body {
 .navbar {
   background: #3071a9;
   border-bottom: 0px;
-  height: 50px;
+  height: 60px;
+  -webkit-box-shadow: 0 8px 6px -6px #999;
```
Comment (1ambda) [inline assets/themes/zeppelin/css/style.css]: 1. Let's unify box-shadow value. 
2. 0px 2px 2px rgba(0, 0, 0, 0.4) would be better. Currently it's too much as @AhyoungRyu mentioned.

File: assets/themes/zeppelin/css/style.css [hunk 3]
```diff
@@ -88,32 +99,41 @@ body {
...
+  box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.4);
 }
 
 .navbar-inverse .navbar-nav > li > a {
+  padding-top: 19px;
+  font-size: 17px;
```
Comment (1ambda) [inline assets/themes/zeppelin/css/style.css]: Let's use smaller font size as @AhyoungRyu mentioned.
15px would be nice.

File: assets/themes/zeppelin/css/style.css (hunk 2 above)
Comment (soralee) [inline assets/themes/zeppelin/css/style.css]: Good! Let's make a decision this! @AhyoungRyu Could you confirm this?

File: assets/themes/zeppelin/css/style.css (hunk 3 above)
Comment (soralee) [inline assets/themes/zeppelin/css/style.css]: @AhyoungRyu Could you confirm this too? :smiley:

File: assets/themes/zeppelin/css/style.css (hunk 2 above)
Comment (1ambda) [inline assets/themes/zeppelin/css/style.css]: One difference between #2371 and this PR is that 

- the background of the main content is white #2371 
- the background color for the main content of the official homepage is blue.

Comment (soralee) [inline assets/themes/zeppelin/css/style.css]: what you say is this line( https://github.com/apache/zeppelin/pull/2371/files#diff-2749d9139fea273c8b492b39bc12eb7dR87)? it looks same color.

**Comment by soralee** [comment]
image

this part is different between #2371. Let me update to match it.

**Comment by 1ambda** [comment]
splitting this PR into multiples 

- might be painful for the author. but it would help to review easily.
- review process might be slower. so hope @AhyoungRyu would help review all separated PRs. If the author split this PR.

**Comment by soralee** [comment]
Um.. So, for quickly review, I'll separate this PR to 2 PR. 
1. updating navbar and main sections. 
2. adding technologies and deployments

**Comment by soralee** [comment]
@1ambda, @Leemoonsoo, @AhyoungRyu, I created a PR regrading `Technologies and Deployments`
#2394. Let me make a remainder PR.

**Comment by 1ambda** [comment]
Additionally, agree with not having too much-shadowed navbar. 
But a small amount of box-shadow would help to distinguish where is the border. 

Here is an example, users can't distinguish the border.

image

**Comment by Leemoonsoo** [comment]
image
Using different background color can be an option, too.

**Comment by soralee** [comment]
I remade a PR about navigation bar and blue section - https://github.com/apache/zeppelin/pull/2401.
Let's move on to 2401 PR. and, I close this issue!
Thanks.

**Pull request closed without merging**